zero/gk_geometry_fromfile.c

```diff
@@ -14,63 +14,44 @@
 
 
 struct gk_geometry*
-gkyl_gk_geometry_fromfile_new(const struct gkyl_rect_grid* grid, const struct gkyl_range *range, const struct gkyl_range* range_ext, 
+gkyl_gk_geometry_fromfile_new(struct gk_geometry* hgeo, const struct gkyl_rect_grid* grid, const struct gkyl_range *range, const struct gkyl_range* range_ext, const struct gkyl_range *global, const struct gkyl_range* global_ext, 
```

Comment:
@akashukla is hgeo meant to be host geo? if so please change to geo_host or geo_ho throughout, which is a convention used elsewhere